pkg/kotsadmconfig/validation/config.go

```diff
@@ -0,0 +1,109 @@
+package validation
+
+import (
+	"github.com/pkg/errors"
+	kotsv1beta1 "github.com/replicatedhq/kots/kotskinds/apis/kots/v1beta1"
+	"github.com/replicatedhq/kots/kotskinds/multitype"
+	configtypes "github.com/replicatedhq/kots/pkg/kotsadmconfig/types"
+	"github.com/replicatedhq/kots/pkg/util"
+)
+
+func ValidateConfigSpec(configSpec kotsv1beta1.ConfigSpec) ([]configtypes.ConfigGroupValidationError, error) {
+	var configGroupErrors []configtypes.ConfigGroupValidationError
+	for _, configGroup := range configSpec.Groups {
+		configGroupError, err := validateConfigGroup(configGroup)
+		if err != nil {
+			return nil, err
```

Comment:
let's wrap the error and tie it to the group name/title for easier troubleshooting/debugging